tracing/src/main/java/io/strimzi/test/tracing/TracingUtil.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test.tracing;
+
+import java.util.List;
+import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.apache.kafka.streams.KafkaClientSupplier;
+
+/**
+ * Tracing initialization
+ */
+public class TracingUtil {
+    private static final Logger log = Logger.getLogger(TracingUtil.class.getName());
```

Comment:
why you don't use log4j as we do in the whole project?

Reply:
OK, changing to log4j.